charts/linkerd2/README.md

```diff
@@ -1,263 +1,124 @@
-# Linkerd2 Helm Chart
+# linkerd2
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![AppVersion: edge-XX.X.X](https://img.shields.io/badge/AppVersion-edge--XX.X.X-informational?style=flat-square)
+
+Linkerd gives you observability, reliability, and security for your microservices — with no code change required.
+
+**Homepage:** <https://linkerd.io>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Linkerd authors | cncf-linkerd-dev@lists.cncf.io | https://linkerd.io/ |
+
+## Source Code
+
+* <https://github.com/linkerd/linkerd2/>
+
+## Requirements
+
+Kubernetes: `>=1.13.0-0`
+
+| Repository | Name | Version |
+|------------|------|---------|
+| file://../add-ons/grafana | grafana | 0.1.0 |
+| file://../add-ons/prometheus | prometheus | 0.1.0 |
+| file://../add-ons/tracing | tracing | 0.1.0 |
+| file://../partials | partials | 0.1.0 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| controllerImage | string | `"ghcr.io/linkerd/controller"` |  |
```

Comment:
How can we keep the descriptions we already had? They have to move as comments in `values.yml`'